mappings/net/minecraft/recipe/ShapedRecipe.mapping

```diff
@@ -10,29 +10,47 @@ CLASS net/minecraft/class_1869 net/minecraft/recipe/ShapedRecipe
 		ARG 2 group
 		ARG 3 width
 		ARG 4 height
-		ARG 5 ingredients
+		ARG 5 input
 		ARG 6 output
-	METHOD method_8145 getPattern (Lcom/google/gson/JsonArray;)[Ljava/lang/String;
+	METHOD method_8145 readPattern (Lcom/google/gson/JsonArray;)[Ljava/lang/String;
 		ARG 0 json
-	METHOD method_8146 combinePattern ([Ljava/lang/String;)[Ljava/lang/String;
-		ARG 0 lines
-	METHOD method_8148 getIngredients ([Ljava/lang/String;Ljava/util/Map;II)Lnet/minecraft/class_2371;
+	METHOD method_8146 removePadding ([Ljava/lang/String;)[Ljava/lang/String;
+		COMMENT
+		COMMENT Removes empty space from around the recipe pattern.
+		COMMENT
+		COMMENT Turns patterns such as:
+		COMMENT "   o"
+		COMMENT "   a"
+		COMMENT "    "
+		COMMENT Into:
+		COMMENT "o"
+		COMMENT "a"
```

Comment:
```suggestion
		COMMENT Turns patterns such as:
		COMMENT <pre>
		COMMENT {@code
		COMMENT "   o"
		COMMENT "   a"
		COMMENT "    "
		COMMENT }
		COMMENT </pre>
		COMMENT Into:
		COMMENT <pre>
		COMMENT {@code
		COMMENT "o"
		COMMENT "a"
		COMMENT }
		COMMENT </pre>
```

Reply:
or maybe `<blockquote>` instead of `{@code }`